### PR TITLE
make AWOA links consistent

### PR DESCRIPTION
**Pull request by cvan**

### CODE REVIEW

**Comment by cvan** [comment]
this is ready to go. lmk if you find any needed changes 😅

File: src/_posts/aframe-v0.3.0.md [hunk 1]
```diff
@@ -9,9 +9,9 @@ image:
 
 [webvr1.0]: https://w3c.github.io/webvr/
 
-A-Frame **v0.3.0** has dropped! With support for the **[new WebVR 1.0
+A-Frame **version 0.3.0** has dropped! With support for the **[new WebVR 1.0
```
Comment (ngokevin) [inline src/_posts/aframe-v0.3.0.md]: `version` a bit inconsistent with previous posts + tag + release notes.

File: src/_posts/aframe-v0.3.0.md [hunk 2]
```diff
@@ -9,9 +9,9 @@ image:
...
 API][webvr1.0]**, we aim higher towards state-of-the-art VR. With room-scale
-tracking, we can walk in a web page, and with tracked controller support, we
+tracking, we can walk in a web page; and with tracked controller support, we
```
Comment (ngokevin) [inline src/_posts/aframe-v0.3.0.md]: semicolons are used for emphasizing the previous clause, no? The tracked controller support is a new clause so seems a comma suffices

**Comment by ngokevin** [comment]
r+wc

File: src/_posts/aframe-v0.3.0.md (hunk 1 above)
Comment (cvan) [inline src/_posts/aframe-v0.3.0.md]: np, it should be obvious enough

File: src/_posts/aframe-v0.3.0.md (hunk 2 above)
Comment (cvan) [inline src/_posts/aframe-v0.3.0.md]: semicolons can be used in two cases: (1) to punctuate two independent clauses and (2) to separate items in lists where commas are being used (e.g., `We have locations in City, State; City, State; and City, State.`)

**Comment by ngokevin** [comment]
merged https://github.com/aframevr/aframe-site/commit/ebfae7f9cec6a19cc97100dbf06fa7a98950a05e

**Pull request closed without merging**